### DISCUSSION TITLE
Can we support multiple archives in a folder?

### DISCUSSION THREAD

**Discussion by RollingStar** [beetbox/beets#4932] [Ideas]
On first glance it doesn't look like it would be that hard to treat archives as folders, and iterate through them the way we do for other folders. Are there any gotchas with doing this? See #4670

https://github.com/beetbox/beets/blob/e10b955a931e4c205b0cadf0860797c0aeee736c/beets/importer.py#L1204

**Comment by cubicvoid** [comment]
I came here to ask about this as well... I've archived my bandcamp collection with https://github.com/easlice/bandcamp-downloader and the directories are all organized by artist and album, but (except for single tracks) the music is all in zip archives. (It's also preferable to keep them that way, since that way the downloader script can just check for new/changed files instead of redownloading everything.) It looks like there might be mild complications with how directories vs archives are currently handled, but probably not too bad. Would a PR implementing this feature be welcome?

**Comment by rcasey212** [comment]
I am new to beets but i find that this could be really useful as well.  What I did see is that at least in Linux you can use the wildcard "*" to import multiple zip files in a directory as it will expand it to individual zip files in that directory.  For example, the following will work:

```$ beet import /path/to/zip_files/*```
or
```$ beet import /path/to/zip_files/*.zip```

This will expand to:
```$ beet import /path/to/zip_files/file_1.zip /path/to/zip_files/file_2.zip path/to/zip_files/file_n.zip```

For now this is working for my purposes.
